cdap-api/src/main/java/co/cask/cdap/api/customaction/AbstractCustomAction.java

```diff
@@ -15,6 +15,9 @@
  */
 package co.cask.cdap.api.customaction;
 
+import co.cask.cdap.api.annotation.TransactionControl;
+import co.cask.cdap.api.annotation.TransactionPolicy;
```

Comment:
Not really. One is an annotation and should not contain other classes. You could, but it is not common practice.